core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -4476,198 +4481,200 @@ private void processO3Block(
+                                            srcDataMax,
+                                            indexBlockCapacity,
+                                            dstFixMem,
+                                            dstVarMem,
+                                            this,
+                                            indexWriter,
+                                            getColumnNameTxn(partitionTimestamp, i)
+                                    );
+                                } catch (Throwable e) {
+                                    if (columnCounter.addAndGet(columnsPublished - columnCount) == 0) {
+                                        o3ClockDownPartitionUpdateCount();
+                                        o3CountDownDoneLatch();
+                                    }
+                                    throw e;
+                                }
+                            }
+
+                            addPhysicallyWrittenRows(srcOooBatchRowSize);
+                        } else {
+                            if (flattenTimestamp && o3RowCount > 0) {
+                                Vect.flattenIndex(sortedTimestampsAddr, o3RowCount);
+                                flattenTimestamp = false;
+                            }
+
+                            // To collect column top values from o3 partition tasks add them to pre-allocated array of longs
+                            // use o3ColumnTopSink LongList and allocate columns + 1 longs per partition
+                            // then set first value to partition timestamp
+                            long colTopSinkIndex = (long) (pCount - 1) * (metadata.getColumnCount() + 1);
+                            long columnTopSinkAddress = colTopSinkIndex * Long.BYTES;
+                            long columnTopPartitionSinkAddr = o3ColumnTopSink.getAddress() + columnTopSinkAddress;
+                            assert columnTopPartitionSinkAddr + (columnCount + 1L) * Long.BYTES <= o3ColumnTopSink.getAddress() + o3ColumnTopSink.size() * Long.BYTES;
+
+                            o3ColumnTopSink.set(colTopSinkIndex, partitionTimestamp);
+                            o3CommitPartitionAsync(
+                                    columnCounter,
+                                    maxTimestamp,
+                                    sortedTimestampsAddr,
+                                    srcOooMax,
+                                    o3TimestampMin,
+                                    o3TimestampMax,
+                                    srcOooLo,
+                                    srcOooHi,
+                                    partitionTimestamp,
+                                    last,
+                                    srcDataMax,
+                                    srcNameTxn,
+                                    o3Basket,
+                                    columnTopPartitionSinkAddr + Long.BYTES
+                            );
+                        }
```

Comment:
There should be `else` which logs a message about how many rows skipped and why in case of ILP 
or throws and exception in case of non-ILP.

Reply:
Inserts will not blow as agreed across all interfaces, so logging is ok, log it as an error

Reply:
done, logging at critical level to get an alert.